source/MaterialXView/Editor.cpp

```diff
@@ -18,11 +18,11 @@ namespace
 class EditorColorPicker : public ng::ColorPicker
 {
   public:
-    EditorColorPicker(ng::Widget* parent, const ng::Color& color) :
+    EditorColorPicker(ng::ref<ng::Widget> parent, const ng::Color& color) :
         ng::ColorPicker(parent, color)
     {
-        ng::Popup* popup = this->popup();
-        ng::Widget* floatGroup = new ng::Widget(popup);
+        ng::ref<ng::Popup> popup = this->popup();
+        ng::ref<ng::Widget> floatGroup = new ng::Widget(popup);
         auto layout = new ng::GridLayout(ng::Orientation::Horizontal, 2,
```

Comment:
I think this `auto` here is going to be a raw pointer too - maybe we should change it as well for consistency?

Reply:
I noticed that one, too, and I think you're right that we should clean up these auto pointers in a subsequent change.